src/rotten-tomatoes/index.js

```diff
@@ -1,3 +1,80 @@
-function rottenTomatoes(grid) {}
+function rottenTomatoes(grid) {
+    let gridCopy = grid
```

Comment:
This does not at all copy the grid. No copy even happens, shallow or otherwise.

For a shallow copy, you'd do

```js
const gridCopy = [...grid]
```

and for a deep copy

```js
const gridCopy = grid.map(field => [...field])
```

But why do you immediately need to make a copy. If the array is really large, this could be a problem.

Reply:
I assumed that, the grid passed to the rottenTomatoes() as argument is a constant and will create a problem when I want to alter values in it, so I made a copy